README.md

```diff
@@ -38,8 +38,8 @@ Blue Button® IG.
 ### Sample Postman Collection
 
 To try out these tests without a CARIN for Blue Button client implementation, you may
-run them using [this Postman collection](https://github.com/inferno-framework/carin-for-blue-button-test-kit/blob/config/C4BB%20Client%20Search%20Tests.postman_collection.json). This Postman collection includes all of the required CARIN for Blue Button profile resource requests and required search
-requests needed to pass all of the tests.
+run them using [this Postman collection](https://github.com/inferno-framework/carin-for-blue-button-test-kit/blob/main/config/C4BB%20Client%20Search%20Tests.postman_collection.json). This Postman collection includes many of the required CARIN for Blue Button profile

```

Comment:
One minor comment, we may want to change this to say all again since the collection now does have all of the required search requests, even if they don't return 200 responses.

Reply:
addressed in [9cface6](https://github.com/inferno-framework/carin-for-blue-button-test-kit/pull/59/commits/9cface6e2b4e0b68c080f41f40c215f9d51abb3c)